wiki/BBCode/en.md

```diff
@@ -1,581 +1,417 @@
+The osu!forums will automatically link pasted URLs inside the post.
+
+- Dedicated button: ![URL button](img/url.jpg "URL button")
+- Notes:
+  - The `LINK` argument does not need quotation marks.
+- Syntax:
+  - Where `LINK` is the url.
+  - Where `TEXT` is the link text.
+```prolog
+[url=LINK]TEXT[/url]
+```
+
+### Profile
+
+This will link to a user profile using the user's name.
+
+Usage of this button is **heavily discouraged**!!
+The problem with using this tag is that user names can be changed once after obtaining an [osu!supporter tag](/wiki/osu!supporter).
+And once they do, the link will fail.
+
+It is recommended to use [URL](#url) instead (using the user's id number).
+
+- Dedicated button: ![Profile button](img/profile.jpg "Profile button")
+- Notes:
+  - Not recommended!
+  - The `USER` argument has to be defined.
+- Syntax:
+  - Where `USER` is the name of the user.
+```prolog
+[profile]USER[/profile]
+```
+
+### Google
+
+This will add a link to a Google Search query using the provided text.
+
+- Dedicated button: ![Google button](img/google.jpg "Google button")
+- Notes:
+  - Know that this will not give everyone the same results.
+    - Some search results are hidden due to language/location.
+- Syntax:
+```prolog
+[google]...[/google]
+```
+
+### Lucky
+
+This will add a direct link to a page using Google's *I'm Feeling Lucky* button using the provided text.
+
+- Dedicated button: ![Lucky button](img/lucky.jpg "Lucky button")
+- Notes:
+  - Know that this will not give everyone the same results.
+    - Some search results are hidden due to language/location.
+- Syntax:
+```prolog
+[lucky]...[/lucky]
+```
+
+### List
+
+This will format the text intro a list.
+
+In BBCode, there are two parts for making the list: the wrapper and the bullets.
+Where the bullets are placed inside the wrapper.
+
+- Dedicated buttons:
+  - List: ![List button](img/list.jpg "List button")
+  - List type: ![List Type button](img/list-type.jpg "List Type button")
+  - List bullet: ![List Bullet button](img/list-bullet.jpg "List Bullet button")
+- Notes:
+  - All bullets must be surrounded by the list/list type tags.
+  - These are the vaild arguments for `TYPE`:
+    - _(empty)_ - bulleted
+    - `1` - numbered
+    - `a` - lettered (lowercased)
+    - `A` - lettered (upper-cased)
+    - `i` - roman numeral (lowercased)
+    - `I` - roman numeral (upper-cased)
+  - By default, bulleted lists uses sqaure bullets.
+  - Lists inside lists are known to be buggy.
+- Syntax:
+  - Where `TYPE` is one of the listed types above.
+  - If `TYPE` is not defined, bullets will be used instead.
+```prolog
+[list=TYPE]
+[*]...
+[/list]
+```
+
+### Image
+
+This will add images to your post.
+The images can be sourced from anywhere, just as long as the image exists from the given url.
+
+It is a nooby mistake to try to link your images directly from your PC.
+Please understand, but using `C:\Users\Alice\Pictures\got-to-see-this.jpg` will **not** work!
```

Comment:
idk, Alice is one of the the fictional examples in computing (along with Bob, and Eve, and a bunch of other names)
https://en.wikipedia.org/wiki/Alice_and_Bob

however, I guess I'll use `Name` instead